samples/sdfw/ssf_client/src/main.c

```diff
@@ -74,14 +38,6 @@ int main(void)
 {
 	LOG_INF("ssf client sample (%s)", CONFIG_BOARD);
 
-	if (IS_ENABLED(CONFIG_ENABLE_ECHO_REQUEST)) {
-		echo_request();
-	}
-
-	if (IS_ENABLED(CONFIG_ENABLE_RESET_EVT_SUBSCRIBE_REQUEST)) {
-		reset_evt_subscribe();
-	}
-
 	if (IS_ENABLED(CONFIG_ENABLE_SDFW_UPDATE_REQUEST)) {
 		sdfw_update();
 	}
```

Comment:
Seems a bit strange to keep the sample here when this is disabled by default and so the sample effectively does nothing. Should we remove the sample entirely? Users should not be using this API for updating SDFW anyway.

Reply:
I agree. If the sample is not changed to do something else with sdfw_services, then it is of no use after removing these services.

Reply:
The only immediate use for this I can see would be for #19278. I still think we can remove `sdfw_update` here though, and even the whole `sdfw_update` client

Reply:
Agreed. Another possible suggestion is to invoke `ssf_psa_generate_random` and get some random values. Given that it is that simple, and no PSA initialization is required.